From: Mingarelli, Thomas <Thomas.Mingarelli@hp.com>
Date: Tue Jun 26 10:27:00 2012 +0200
Subject: hpwdt: Unregister NMI events on exit.
Git-commit: a089361cf5f1d6a5295aa5385238bd044998e1e9
Patch-mainline: v3.5-rc5
References: bnc#777746

watchdog: hpwdt: Unregister NMI events on exit.
    
This patch is to unregister for NMI events upon exit. Also we are now
making the default setting for allow_kdump enabled.

NOTE: I'm only including the changed default for allow_kdump.

Signed-off-by: Thomas Mingarelli <thomas.mingarelli@hp.com>
Signed-off-by: Wim Van Sebroeck <wim@iguana.be>

Signed-off-by: Petr Tesarik <ptesarik@suse.cz>

---
 drivers/watchdog/hpwdt.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/watchdog/hpwdt.c
+++ b/drivers/watchdog/hpwdt.c
@@ -143,7 +143,7 @@ struct cmn_registers {
 }  __attribute__((packed));
 
 static unsigned int hpwdt_nmi_decoding;
-static unsigned int allow_kdump;
+static unsigned int allow_kdump = 1;
 static unsigned int priority;		/* hpwdt at end of die_notify list */
 static unsigned int is_icru;
 static DEFINE_SPINLOCK(rom_lock);
